CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+## [1.5.0] - 2017-08-18
```

Comment:
Ideally this should have been omitted as well but I will merge this anyways since everything else is right now.